index.html

```diff
@@ -92,11 +91,14 @@ <h1>Work Experience</h1>
             <h2>Staff Software Engineer - Optimizely</h2>
             <p class="subDetails">August 2020 - Present</p>
             <ul class="experienceList">
-              <li>Tech Lead for the Account Services Team, responsible for the technical roadmap and execution of an identity migration from a homegrown/Auth0 hybrid solution to Okta</li>
-              <li>Championed a solution for authentication and authorization outside of our monolith, unblocking microservice development and accelerating the entire engineering organization</li>
-              <li>Contributed to our in-house deployment platform (on Kubernetes), and drove adoption throughout the organization via consulting and pair-programming</li>
```

Comment:
No parentheses around "on Kubernetes". Maybe reword to "our in-house, Kubernetes-based deployment platform"